docs/Integration_Guide.md

```diff
@@ -0,0 +1,115 @@
+The Process of Spherex Protection into the smart contracts is usually done (at least at this point) by the tech team at Spherex.
+
+That being said, and because the developer of the smart contracts will always know it better than we do, it is important for you as the developer of the protected contract to undestand the process of integration and the various aspects of that process.
+
+- [Protection  - the big picture](#protection----the-big-picture)
+- [Some terms and defenitions](#some-terms-and-defenitions)
+    - [Contracts in the system](#contracts-in-the-system)
+    - [Entities and addresses](#entities-and-addresses)
+- [Integration](#integration)
+  - [Inline vs Proxy](#inline-vs-proxy)
+  - [Configuration](#configuration)
+  - [Deployment](#deployment)
+    - [Special cases](#special-cases)
+- [What Should you do?](#what-should-you-do)
+  - [Roles and Permissions](#roles-and-permissions)
+  - [For each of you'r contracts:](#for-each-of-your-contracts)
+    - [For immutable contracts (inline protection):](#for-immutable-contracts-inline-protection)
+    - [For mutable contracts (proxy protection):](#for-mutable-contracts-proxy-protection)
+    - [In the deployment Scripts](#in-the-deployment-scripts)
+
+
+## Protection  - the big picture
```

Comment:
I think this is redundant as it is already illustrated in the docs

Reply:
I don't want to assume the devs read the docs **and remember it**